articles/embedding-a-video-into-a-webpage-using-html-and-css/Index.md

```diff
@@ -0,0 +1,139 @@
+
+![step 7](/engineering-education/embedding-a-video-into-a-webpage-using-html-and-css/step-7.jpg)
+
+**Step 8:** Save the changes you made on your HTML page. 
+After saving, while you are still on your HTML page, right click and select “Open in Browser”. By clicking on Open in Browser, your HMTL code will run in your browser.
+
+![step 8](/engineering-education/embedding-a-video-into-a-webpage-using-html-and-css/step-8.jpg)
+
+**Step 9:** Your Browser’s output.
+Depending on your HTML code(on how you have styled it), you can place the iframe tag anywhere you want within your HTML page. After you have successfully embedded a video in your webpage, you can actively interact with it meaning, you can fast forward and rewind it, play and pause, increase, and decrease the volume or even mute, click on YouTube icon in the embedded page to play the video directly on YouTube etc. All those activities are called video controls.
+
+![step 9](/engineering-education/embedding-a-video-into-a-webpage-using-html-and-css/step-9.jpg)
+
+**Step 10:** Enjoy your view.
+As mentioned earlier, CSS is basically for styling your webpage. Like the above screenshot contains a brown-ish/red-ish background and the screenshot below contains a purple background. You can play with the HTML and CSS skills to suit your preferences.
+ 
+ ![step 10](/engineering-education/embedding-a-video-into-a-webpage-using-html-and-css/step-10.jpg)
+
+### Important Tips
+- Avoid hyperlinked videos: The URL’s Video should be on its own without any characters or spaces before or after the URL and should not be a hyperlinked or it will not be embedded. What is hyperlink? is a highlight done to a link such that when it is clicked or hovered, the browser automatically displays another page or changes the current page to show the referenced content. 
+- You can embed images and other contents as well into a post or page as long as the item you want to embed has its own URL.
+- Due to copyrights, if the video you want to embed is not your own creation, it is prudent to ask permission from the owner.
+
+### Conclusion 
+Along with knowing how to embed a video using HTML and CSS, we have learned the reasons behind using iframe tag and URL rather than ``` <video>```  tag in HTML5.
+
+To summarize:
+- We have gained the knowledge on how to embed videos.
+- we have gained the knowledge on how to also embed other items like images using the same steps.
+- We have learned the benefits of embedding a video into a web page.
+- We now know what HTML, CSS, URL, IFRAME are.
+- We have learnt the things we should avoid while embedding, such as hyperlinks.
+- We have learned that there are other social media platforms that supports videos apart from YouTube.
+- We have learnt the important tips that we should know or take before embedding other creator’s content into a different webpage.
+- We have learnt the tools to bag for a successful embedding.
+
```

Comment:
To summarize:
- We have understood the concept of video embedding.
- We have gained an overview of HTML, CSS, and Iframe.
- We have known things we should avoid when embedding videos.
- We have learned how to embed videos.
- We have learned important tips that we should know or consider before embedding videos.